### ISSUE TITLE
host_url was removed?

### ISSUE THREAD

**Issue by fama** [amirali/kutt#12]
The host_url seems to have been removed, and now this python wrapper is broken.  :-(

https://github.com/amirali/kutt/blob/08fe5afb8e086d5aa64ee046ca4dfcbe1df6012e/kutt/kutt.py

**Comment by amirali** [comment]
It got fixed in v1.5.1.

**Comment by fama** [comment]
I just tried it with the latest version (kutt==1.5.1) and it's broken  :-(

kutt==1.3.3 works fine...

Sorry for reopening the issue.


```log
Traceback (most recent call last):
  File "/home/user/.virtualenvs/myproject/lib/python3.10/site-packages/requests/models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
  File "/home/user/.virtualenvs/myproject/lib/python3.10/site-packages/simplejson/__init__.py", line 525, in loads
    return _default_decoder.decode(s)
  File "/home/user/.virtualenvs/myproject/lib/python3.10/site-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/home/user/.virtualenvs/myproject/lib/python3.10/site-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/user/it-python/myproject/urls2microsip.py", line 192, in <module>
    df = get_short_urls(df)
  File "/home/user/it-python/myproject/urls2microsip.py", line 70, in get_short_urls
    kutt.links(config['kutt']['api'],
  File "/home/user/.virtualenvs/myproject/lib/python3.10/site-packages/kutt/kutt.py", line 45, in links
    data = res.json()
  File "/home/user/.virtualenvs/myproject/lib/python3.10/site-packages/requests/models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
